build.gradle.kts

```diff
@@ -45,9 +46,11 @@ val argbash by configurations.creating
 val commonComponents by configurations.creating
 val mavenComponents by configurations.creating
 
+val buildScanSummaryVersion = "1.0-2023.2"
```

Comment:
Move this line right below `distributionVersion` on line 39.

Reply:
Done in https://github.com/gradle/gradle-enterprise-build-validation-scripts/pull/487/commits/b9cb2f603e6c14a588d1b864c12ee0459872037f